bower.json

```diff
@@ -23,7 +23,15 @@
     "foundation": "~5.5.2",
     "foundation-datepicker": "~1.3.0",
     "angular-loading-bar": "~0.8.0",
-    "raven-js": "~1.3.0"
+    "iron-ajax": "PolymerElements/iron-ajax#~1.1.1",
```

Comment:
could we describe what feature requires certain dependencies? it would be easier to manage/remove them in the future. For example, is every above dependency necessary (maybe it was required by old autocomplete)?